### ISSUE TITLE
WS280 41: Some VC2010 transcripts are missing a parent, is this important?

### ISSUE THREAD

**Issue by MagdalenaZZ** [WormBase/wormbase-pipeline#159]
Issue: Some VC2010 transcripts are missing a parent. In the build, the issue was fixed with a patch

If we continue to use the current VC2010 database, the patch is applied. However, anytime we re-build VC2101 the issue will re-emerge.

Action:  find out the importance of the missing gene spans in the original GFFs, and decide if we need to have a script to add them in if they are missing.

**Comment by scottcain** [comment]
Just for future reference, it's important to me (that is, to JBrowse).

**Comment by Paul-Davis** [comment]
I don't think this is a WS release issue more an issue for when we regenerate and VC2010 files in the future.

**Comment by MagdalenaZZ** [comment]
Yes @scottcain , currently it will work fine. It is just if we re-generate the lift-over sometime in the future this will become an issue again, unless we do something about it.
